bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -298,7 +298,7 @@ spec:
           path: /var/lib/openvswitch/data
       - name: etc-openvswitch
         hostPath:
-          path: /var/lib/openvswitch/etc
```

Comment:
I'm not sure if we need this mount for the `ovsdb system id` as stated in the comment atop the daemonset yaml  @abhat Do you know if this mount is used anymore? (I didn't see anything there on the host last time I looked)

Reply:
@astoycos  FWIW when I checked the folder in openshift cluster it was empty that is why I try to reuse it but yeah if I missed anything I can create new mount

Reply:
Yep I saw the same thing... @abhat Any Idea if we can repurpose this mount here?

Reply:
If you're on a cluster, you should be able to "ps ax | grep ovs" and see where ovsdb-server on the host puts its db. That's very likely going to be /etc/openvswitch. I think at some past point it used to be in /var/lib/openvswitch/etc but it's just /etc now, I'm 99% sure.

Reply:
But also, ovnkube node gets the system-id from `ovs-vsctl` these days, so I can't think why we'd need direct access to the OVS database. Instead we should just be talking to the dbserver socket.

Reply:
> If you're on a cluster, you should be able to "ps ax | grep ovs" and see where ovsdb-server on the host puts its db. That's very likely going to be /etc/openvswitch. I think at some past point it used to be in /var/lib/openvswitch/etc but it's just /etc now, I'm 99% sure.

yes Dan if I login w/o chroot its under /host/etc/openvswicth if I ran as root and this what ovn node runs as its under /etc/openvswitch I did test it and I showed the content after modify the mount host path

Reply:
what happens if we just remove this mount? is it even needed at all?

Reply:
oh duh, nevermind. ignore this.